### DISCUSSION TITLE
Metrics for secret scanning with GHAS

### DISCUSSION THREAD

**Discussion by 15MariamS** [community/community#51595] [Enterprise]
### Select Topic Area

Product Feedback

### Body

If you have GitHub Advanced Security:
- You can now view alert metrics for your custom patterns defined at the repository, organization, and enterprise levels directly from the custom pattern's page.
- For any custom pattern that has push protection enabled, you can also view push protection metrics like total secrets blocked and bypassed.

We'd love your feedback on our metrics and what else you might want to see to help you measure secret scanning's impact! Leave your thoughts below 💭

**Comment by samigt** [comment]
Love the Successful blocks metric !! It will be great if there is a view to see successful blocks across all secrets in one view.

Great Work !!

**Reply by 15MariamS** [reply]
Woo! That's coming soon too @samigt 🙌
